src/index.js

```diff
@@ -1,18 +1,59 @@
 import './styles.css';
 import UI from './ui';
-
-const todos = [
-  {
-    description: 'Mow lawn',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Clean bathroom',
-    completed: true,
-    index: 5,
-  },
-];
+import validateDescription from './validateDescription';
 
 // When content loads
-document.addEventListener('DOMContentLoaded', UI.addApp(todos));
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', UI.addApp());
+
+// Event: when form is submitted
+document.querySelector('#addTaskForm').addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  // Get form values
+  const taskDescription = document.querySelector('#taskDesc').value.trim();
+  if (!validateDescription(taskDescription)) {
+    console.log('Error - empty description');
+  } else {
+    // Add task
+    UI.addTaskStore(taskDescription);
+    // reloads page
+    // eslint-disable-next-line no-restricted-globals
+    location.reload();
+  }
+});
+
+// Event: when icon is clicked to add task
+document.querySelector('#clickEnterIcon').addEventListener('click', () => {
+  // Get form values
+  const taskDescription = document.querySelector('#taskDesc').value.trim();
+  if (!validateDescription(taskDescription)) {
+    console.log('Error - empty description');
+  } else {
+    // Add task
+    UI.addTaskStore(taskDescription);
+
+    // reloads page
+    // eslint-disable-next-line no-restricted-globals
```

Comment:
- [ ] Please remove this eslint disable line it's best to fix the issue in the code.